.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,8 +40,11 @@ You can use the following command to let the DevOps SIG members help you to revi
 /cc @kubesphere/sig-devops 
 And please avoid cc any individual.
 -->
-```
-```
+Please check the following list before waiting reviewers:
+
+- [ ] Already committed the CRD files to [the Helm Chart](https://github.com/kubesphere-sigs/ks-devops-helm-chart/) if you created some new CRDs
+- [ ] Already [added the permission](https://github.com/kubesphere/ks-installer/blob/9e063b085a0e43fdb3d0d9e3e7f4149146f14b9c/roles/ks-core/prepare/files/ks-init/role-templates.yaml) for the new API
+- [ ] Already added the RBAC annotations for the new controllers
```

Comment:
IMO, `RBAC markers` is more suitable than `RBAC annotations` here.

 See https://kubebuilder.io/reference/markers/rbac.html?highlight=rbac#rbac for more.